ocrd_network/ocrd_network/logging.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from ocrd_utils import safe_filename
+
+
+OCRD_NETWORK_MODULES = [
+    "mets_servers",
+    "processing_jobs",
+    "processing_servers",
+    "processing_workers",
+    "processor_servers"
+]
+
+
+def get_root_logging_dir(module_name: str) -> Path:
+    if module_name not in OCRD_NETWORK_MODULES:
+        raise ValueError(f"Invalid module name: {module_name}, should be one of: {OCRD_NETWORK_MODULES}")
+    # TODO: Utilize env variable to set the root
```

Comment:
I thought about that too, not only for logging but for the mets server too. I think this could be part of the PR